src/apis/hermez-web.ts

```diff
@@ -1,17 +1,31 @@
 import axios from "axios";
+import * as z from "zod";
 
 import { HERMEZ_WEB_URL } from "src/constants";
 
 const baseApiUrl = HERMEZ_WEB_URL;
 
+interface WalletUnderMaintenanceResponse {
+  isWalletUnderMaintenance: number;
+}
+
+const walletUnderMaintenanceResponseParser: z.ZodSchema<WalletUnderMaintenanceResponse> = z.object({
```

Comment:
Shouldn't we move this `walletUnderMaintenanceResponseParser` to the persistence module and `WalletUnderMaintenanceResponse` to the domain?

Reply:
As [we agreed here](https://github.com/hermeznetwork/wallet-ui/pull/635#discussion_r701202023) everything under `/apis` needs to be moved to the `/persistence` namespace. I would though first migrate to TS and then reorganize the modules. Does it make sense?

> and WalletUnderMaintenanceResponse to the domain

I don't think the interfaces that represent responses from APIs belong to the `domain`. I think the domain layer should only include first-class interfaces that apply downward throughout the whole application.

Reply:
Right! I forgot about it. Both things makes sense to me